performance-tests/bench_2/test_controller/ScenarioManager.cpp

```diff
@@ -354,6 +354,10 @@ void ScenarioManager::execute(const Bench::TestController::AllocatedScenario& al
             if (json_2_idl(ss, report)) {
```

Comment:
This `report` variable has the same name as the last parameter of this function. It's not a problem but could it be renamed to improve readability?